identity/src/main/java/com/android/identity/credentialtype/Options.kt

```diff
@@ -0,0 +1,493 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android.identity.credentialtype
+
+/**
+ * Object containing reusable lists of [IntegerOption] or [StringOption]
+ */
+object Options {
```

Comment:
This is helpful but it's a lot of data and I'm concerned some downstream consumers will not like it. Please move to `knowntypes` package so it'll be easier to split into a separate `identity-knowntypes` library in some point in the future (to save space).

Reply:
Done